src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -18,6 +18,7 @@ Object {
       "fair-deal": "#EAA403",
```

Comment:
I can't comment on the empty file above, so this one is out of place :D
care to delete the `Loading` file above? looks like an accident

Reply:
Huch